### PR TITLE
Migrate CardTemplate to Kotlin

### PR DESCRIPTION
**Pull request by ShridharGoel**
Migration of CardTemplate to Kotlin.

### CODE REVIEW

**Comment by david-allison-1** [comment]
Blocked on https://github.com/ankidroid/Anki-Android/pull/9480

File: AnkiDroid/kotlinMigration.gradle [hunk 1]
```diff
@@ -38,7 +38,7 @@ permission notice:
     DEALINGS IN THE SOFTWARE.
 */
 
-def className = ""
+def className = "com/ichi2/anki/cardviewer/CardTemplate.kt"
```
Comment (david-allison-1) [inline AnkiDroid/kotlinMigration.gradle]: This doesn't compile. 

It should be `/com/ichi2/anki/cardviewer/CardTemplate.kt`

See: https://github.com/david-allison-1/Anki-Android/commit/632a20741914189a319d2e55bb5fef30a9fd2216#diff-4abf8497cb1570dab86199e9f63e1c416d78e9dc08341af3b96ae0f485bf530dR1

We should also add an example on the line above to ensure this doesn't happen again

**Review by david-allison-1 — CHANGES_REQUESTED** [review]

File: AnkiDroid/kotlinMigration.gradle (hunk 1 above)
Comment (ShridharGoel) [reply AnkiDroid/kotlinMigration.gradle]: Updated.

**Review by david-allison-1 — APPROVED** [review]
LGTM

**Review by mikehardy — APPROVED** [review]
:steam_locomotive:

**Pull request merged**